src/components/progress-circle/progress-circle.scss

```diff
@@ -0,0 +1,97 @@
+@import "variables";
+
+@import "default-theme";
+
+
+$circle-ease-in-out : cubic-bezier(0.35, 0, 0.25, 1) !default;
```

Comment:
All of the vars should be prefixed with `md-` so that people can combine them with their own scss builds eventually.